models/surgical_tool_segmentation/docs/readme.md

```diff
@@ -0,0 +1,95 @@
+# Description
+A pre-trained model for the surgical tool segmentation task.
+
+# Model Overview
+This model is trained using a flexible unet structure with an efficient-b0 [1] as the backbone and a UNet architecture [2] as the decoder. Datasets use private samples from [Activ Surgical](https://www.activsurgical.com/).
+The [pytorch model](https://drive.google.com/file/d/14r6WmzaZrgaWLGu0O9vSAzdeIGVFQ3cs/view?usp=sharing) and [torchscript model](https://drive.google.com/file/d/1i-e5xXHtmvmqitwUP8Q3JqvnmN3mlrEm/view?usp=sharing) are shared in google drive. Details can be found in large_files.yml file.
```

Comment:
Suggest to add note: 

Modify the path specified in configs/train.json: `"bundle_root"` to reflect where the models are downloaded. Under the path specified by `"bundle_root"` the expected directory structure is `models/model.pt`.

Reply:
Hi @jin-nvidia ,
     Thank you for your advice. This note has been added to Model Overview part.
Thanks,
Bin